song-server/src/test/java/org/icgc/dcc/song/server/service/UploadServiceTest.java

```diff
@@ -54,82 +54,63 @@
   @Autowired
   UploadService uploadService;
 
-  @SneakyThrows
-  private String readFile(String name) {
-    return new String(Files.readAllBytes(new java.io.File("..", name).toPath()));
-  }
-
   @Test
   @SneakyThrows
   public void testUploadSequencingRead() {
-    val json = readFile("sequencingRead.json");
-    testUpload(json);
+    test("sequencingRead.json");
   }
 
   @Test
   @SneakyThrows
   public void testUploadVariantCall() {
```

Comment:
Change method name to "testVariantCall". If you want to couple the method name with the intent, then rename to "testUploadAndSaveVariantCall". Like wise for testUploadSequencingRead